style/facelift.scss

```diff
@@ -0,0 +1,242 @@
+$font-newspaper: georgia, 'PT Serif', serif;
+$font-ui: 'Avenir Next', 'Montserrat', sans-serif;
+$transition-fast: 0.3s ease;
+$offblack : #333;
+$offwhite : #ededed;
+$accent1  : #F16061;
+$accent2  : #FDD484;
+$accent3  : #AFD792;
+$accent4  : #a8d2bf;
+
+
+@mixin newspaper {
+  // text-indent: 20px;
+  // Think about em-dashes and more spaces / automatic <p> creation
+  white-space: pre-wrap;
+  line-height: 1.5;
+  font-family: $font-newspaper;
+}
+@mixin ul-reset {
+  list-style: none;
+  padding-left: initial;
+  margin-bottom: initial;
+}
+@mixin vertical-align($position: relative) {
+  position: $position;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.text-wrapper {
+  max-width: 500px;
+  margin: 0 auto;
+  /* Make selection not span the whole page, just this div */
+  position: relative;
+
+  &__text {
+    @include newspaper;
+
+    /* the annotation plugin changes manually the color and
+       structure of the page so this doesn't work */
+    &[data-topic="1"]::selection {
+      background: $accent1;
+    }
+    &[data-topic="2"]::selection {
+      background: $accent2;
+    }
+    &[data-topic="3"]::selection {
+      background: $accent3;
+    }
+    &[data-topic="4"]::selection {
+      background: $accent4;
+    }
+  }
+}
+
+.topic-picker {
+  $nav-width: 25%;
+
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 10px;
+
+  background-color: $offblack;
+  overflow: hidden;
+
+  transition: background $transition-fast,
+              height $transition-fast;
+
+  &__nav {
+    @include ul-reset;
+    $height: 30px;
```

Comment:
and if it's something where height is a relative thing, you can always just do math `$height * 1.5`, etc.